code/game/atoms.dm

```diff
@@ -58,6 +58,9 @@
 /atom/proc/CheckParts()
 	return
 
+/atom/proc/activate()

```

Comment:
Does this really need to be on /atom?
It looks like the kind of stuff that would make more sense on /obj/item